networking/network_observability/installing-operators.adoc

```diff
@@ -1,13 +1,22 @@
 :_content-type: ASSEMBLY
 [id="installing-network-observability-operators"]
-= Installing Network Observability operators in {product-title}
+= Installing the Network Observability operator
 include::_attributes/common-attributes.adoc[]
 :context: network_observability
 
 toc::[]
+Installing the Loki Operator is a  prerequisite for using the Network Observability operator. The Loki Operator integrates a gateway that implements multi-tenancy & authentication with Loki for logging. The Network Observability Operator itself is not (yet) multi-tenant. 
```

Comment:
Could we change this a little bit to say **Loki** is a prerequisite, and we recommend installing the **Loki Operator** ? 
This is more or less the same discussion I had with @xenolinux here: https://github.com/openshift/openshift-docs/pull/53360/files#diff-f1449138f2d1175df0811b265917416523d8c44352f06cdb14b13a11db3fca11R16
There are several ways to install Loki and we won't enforce a single one, but it's a recommendation.

Reply:
I instead tweaked the information here and also in modules/network-observability-loki-install.adoc